src/typings/lead.tsx

```diff
@@ -0,0 +1,58 @@
+export type Confidentiality = 'confidential' | 'unprotected';
+export type LeadStatus = 'pending' | 'processed';
+
+export interface DatabaseEntityBase {
+    id: number;
+    createdAt: string;
+    createdBy: number;
+    createdByName: string;
+    modifiedAt: string;
+    modifiedBy: number;
+    modifiedByName: string;
+    versionId: number;
+}
+
+export interface UserDetails {
+    id: number;
+    email: string;
+    displayName: string;
+    displayPicture?: number;
+}
+
+export interface OrganizationDetail {
+    id: number;
+    title: string;
+}
+
+export interface Lead extends DatabaseEntityBase {
+    title: string;
+    assessmentId?: number;
+    assignee?: number;
+    assigneeDetails: UserDetails;
+    confidentiality: Confidentiality;
+    priority: number;
+    priorityDisplay: string;
+    project: number;
+    publishedOn: string;
+    status: LeadStatus;
+    sourceType: string;
+
+    source: number;
+    sourceRaw?: string;
+    sourceDetail: OrganizationDetail;
+    athors?: [number];
```

Comment:
This should be a typo
```suggestion
    authors?: [number];
```